### PR TITLE
Show how lemma is used in a specific verse + search for reference

### PR DESCRIPTION
**Pull request by welcome-me**
I think this is at least 80% of the way there. However, there are a couple things I noticed/ know could be improved.

- ~~Downloading the **entire** byztext to look up one or two verses is inefficient. Perhaps, it gets cached after the first page load, but it might be better to store the verses in individual files.~~ Books are now lazy loaded as needed. (This would also make it easy to add additional sources / translations -- we English only folks would like to be able to look up verses in an English translation and see what word it is in Greek.)
- ~~It highlights OT verses which are not found in the Byztext.~~
- *Data Integrity:* Some of the `OsisRef` tags in the lexicon have incomplete or invalid data.

**Notice:** I got excited and added searching for specific verses.

### CODE REVIEW

File: simple.css [hunk 1]
```diff
@@ -947,27 +947,60 @@ etym, re {
...
-#verse-popup span.highlighted {
+#verse-popup .word.highlighted {
   color: var(--accent);
   font-weight: bold;
+}
+
+#verse-popup .bottomBar {
+  display: flex;
+  flex-direction: row;
+  text-align: center;
```
Comment (mrgreekgeek) [inline simple.css:1002]: We should probably add `padding: .25em;` to .bottomBar to make it match the "words" div padding.

**Review by mrgreekgeek — COMMENTED** [review]
This is a _really_ neat addition that makes the popup even more useful! Awesome idea.

File: index.html
Comment (mrgreekgeek) [inline index.html:1]: This is a _really_ neat addition that makes the popup even more useful! Awesome idea.

**Comment by mrgreekgeek** [comment]
Wow, thanks so much, this is some really good work! 

> * Downloading the **entire** byztext to look up one or two verses is inefficient. Perhaps, it gets cached after the first page load, but it might be better to store the verses in individual files. (This would also make it easy to add additional sources / translations -- we English only folks would like to be able to look up verses in an English translation and see what word it is in Greek.)

Yes, I know. :( Individual files seems a bit too fragmentary, because then we'd literally have thousands of files floating around. At that point we'd be basically writing a Bible API, wouldn't we? That would be sorta like reinventing the wheel, cause there are surely APIs that already do this. What about downloading only the Bible book that is needed for a particular reference, rather than the whole NT all at once? 

Have you seen https://fetch.bible/access/enhancer/ ? The API is already built, but doesn't seem to currently support Strongs numbers. I wonder if we'd be better off enhancing that one to support Strongs. By doing so we could improve an existing free app rather than starting from scratch. Plus then it would be a cinch to add other Greek texts/English translations to the mix from their existing collection of texts that are available from the API. 

We could also definitely save some bytes by minifying the JSON file too; it's got a lot of whitespace right now. We're also sending along parsing codes that we don't use (yet).

File: index.html
Comment (welcome-me) [reply index.html:1]: @mrgreekgeek What? The comment is on the entire file, and I'm not sure what you're congratulating me on. 🫣😊

Comment (mrgreekgeek) [reply index.html:1]: > @mrgreekgeek What? The comment is on the entire file, and I'm not sure what you're congratulating me on. 🫣😊

Sorry, I commented on commit 49872334e699331a110660c92651a6b5df6b7f1e and thought my comment would be attached to it. I was particularly impressed with the feature that lets a user click on _any_ word in the verse to load the definition. Really handy!

Comment (welcome-me) [reply index.html:1]: Thanks for passing on your approval. :)

**Comment by welcome-me** [comment]
Things that need done, though they probably don't all prevent merging what we have. 

1. ~~Displaying what text is being used.~~
2. ~~No highlight Old Testament~~
3. ~~Fix Revelation~~
4. ~~Lazy load books instead of downloading the entire New Testament on page load.~~
    - ~~show loading indicator when a reference is being looked up~~
5. ~~Normalize osis references in lexicon.~~
6. Support showing ranges in verse popup

**Comment by welcome-me** [comment]
@mrgreekgeek I'm ready for you to re-consider merging this into main... I'm not too concerned with adding verse ranges soon, and this makes several functionality improvements over the currently deployed app.

**Comment by mrgreekgeek** [comment]
> @mrgreekgeek I'm ready for you to re-consider merging this into main... I'm not too concerned with adding verse ranges soon, and this makes several functionality improvements over the currently deployed app.

Thank you SO much for all the work you put into this, @welcome-me! I agree, let's get these improvements out there, and worry about other issues in future updates.

**Review by mrgreekgeek — APPROVED** [review]

**Pull request merged**